core/load-git.cpp

```diff
@@ -357,22 +357,28 @@ static char *parse_keyvalue_entry(void (*fn)(void *, const char *, const std::st
 		line++;
 	}
 
-	if (c == '=')
+	if (c != 0)
 		*line++ = 0;
 
-	char *start_val = line;
-	while ((c = *line) != 0) {
-		if (isspace(c))
-			break;
-		line++;
-	}
+	std::string val;
+	if (c == '=') {
+		char *start_val = line;
```

Comment:
I understand that you are just using old code here. But is there any reason that this isn't `const char *`?

Reply:
Copy / pasta.